File: petclinic-frontend/src/features/veterinarians/models/VetResponseModel.ts
import { Workday } from './Workday.ts';
import { Speciality } from './Speciality.ts';

export interface VetResponseModel {
  vetId: string;
  vetBillId: string;
  firstName: string;
  lastName: string;
  email: string;
  phoneNumber: string;
  resume: string;
  workday: Workday[];
  workHoursJson: string;
  active: boolean;
  specialties: Speciality[];
  password: string; // Add this line
  username: string;
}

File: billing-service/src/main/java/com/petclinic/billing/domainclientlayer/models/VetResponseModel.java
package com.petclinic.billing.domainclientlayer.models;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class VetResponseModel {
    private String vetId;
    private String vetBillId;
    private String firstName;
    private String lastName;
    private String email;
    private String phoneNumber;
//    private String resume;
//    //private Set<Workday> workday;
//    private boolean active;
//    private Set<SpecialtyDTO> specialties;
}
